packages/uiweb/src/lib/components/notification/chainDetails.tsx

```diff
@@ -1,15 +1,20 @@
 import EthereumSVG from "../../icons/ethereum.svg";
 import PolygonSVG from "../../icons/polygon.svg";
 import GraphSVG from "../../icons/thegraph.svg";
+import Tooltip from "../tooltip";
 
-const createSVGIcon = (url: string, alt: string) => {
-  return <img src={url} alt={alt} />
+const createSVGIcon = (url: string, chainName: string) => {
+  return (
+    <Tooltip tooltipContent={`Delivered by ${chainName}`}>
+      <img src={url} alt={chainName.toUpperCase()} />
+    </Tooltip>
+  )
 }
 
 export default {
-    ETH_TEST_GOERLI: { label: "ETHEREUM GOERLI", icon: createSVGIcon(EthereumSVG, "ETHEREUM GOERLI") },
-    ETH_MAINNET: { label: "ETHEREUM MAINNET", icon: createSVGIcon(EthereumSVG, "ETHEREUM MAINNET") },
-    POLYGON_TEST_MUMBAI: { label: "POLYGON MUMBAI", icon: createSVGIcon(PolygonSVG, "POLYGON MUMBAI") },
-    POLYGON_MAINNET: { label: "POLYGON MAINNET", icon: createSVGIcon(PolygonSVG, "POLYGON MAINNET") },
-    THE_GRAPH: { label: "THE GRAPH", icon: createSVGIcon(GraphSVG, "THE GRAPH") },
-};
\ No newline at end of file
+    ETH_TEST_GOERLI: { label: "ETHEREUM GOERLI", icon: createSVGIcon(EthereumSVG, "Ethereum Goerli") },
+    ETH_MAINNET: { label: "ETHEREUM MAINNET", icon: createSVGIcon(EthereumSVG, "Ethereum Mainnet") },
+    POLYGON_TEST_MUMBAI: { label: "POLYGON MUMBAI", icon: createSVGIcon(PolygonSVG, "Polygon Mumbai") },
```

Comment:
why was the "ETHEREUM GOERLI" changed to "Ethereum Goerli"

Reply:
Actually, the text we want to display on the tooltip is in title case (first letter capital of each word), so, I changed the titles to title case and passed them to the function then I used toUpperCase() for alt ppt of the image. Can do it the other way as well, ie to keep the func parameters in upper case and write small logic to convert it to title case, thought that would involve unnecessary processing.

Reply:
okay got it